docs/concepts/node-media.md

```diff
@@ -62,8 +62,7 @@ This migration is performed by all installation methods, so these values should
 be available "out of the box". These values were provided by the PCDM data model
 (see [RDF in Islandora](rdf.md)),
 and not all values are associated with behaviours you might expect - for instance, 
-we do not make transcripts show up by tagging a file with "Transcript". See 
-[transcripts](../user-documentation/transcripts.md) instead.
+we do not make transcripts show up by tagging a file with "Transcript". 
```

Comment:
Islandora is not configured out-of-the box to display a transcript if you add a Media (file) and tag it as "Transcript".